package/boot/uboot-rockchip/src/configs/xiguapi-v3-rk3568_defconfig

```diff
@@ -0,0 +1,88 @@
+CONFIG_ARM=y
```

Comment:
again, please refresh u-boot defconfig. if you'd bother/don't know how to do that, use [this](https://github.com/u-boot/u-boot/blob/v2025.10/configs/lubancat-2-rk3568_defconfig) as a template